src/dense/generic_dense.jl

```diff
@@ -361,6 +361,9 @@ function ode_interpolant(Θ,dt,y₀,y₁,k,cache::OrdinaryDiffEqMutableCache,idx
   if typeof(idxs) <: Number || typeof(y₀) <: Union{Number,SArray}
     # typeof(y₀) can be these if saveidxs gives a single value
     _ode_interpolant(Θ,dt,y₀,y₁,k,cache,idxs,T)
+  elseif typeof(idxs) <: Nothing
+    out = oneunit(Θ) .* y₁
+    _ode_interpolant!(out,Θ,dt,y₀,y₁,k,cache,idxs,T)
   else
     # determine output type
```

Comment:
Couldn't we just change the logic in this branch instead of adding a new one? At least it seems that the `if typeof(idxs) <: Nothing` checks in this third branch could be removed.

Reply:
I was triggering tests to see if this would work, but yes y1[idxs] would be a simpler way to do the other branch. Looks like it's passing now